src/page-objects/AdministrationPages.ts

```diff
@@ -21,6 +24,9 @@ export interface AdministrationPageTypes {
     AdminFlowBuilderDetail: FlowBuilderDetail;
     AdminDataSharing: DataSharing;
     AdminDashboard: Dashboard;
+    AdminCategories: AdminCategories;
+    AdminLandingPageCreate: AdminLandingPageCreate;
+    AdminLandingPageDetail: AdminLandingPageDetail;
```

Comment:
The class still can hold the name..like Categories.ts ...just the Reference should have the prefix of Admin